package.json

```diff
@@ -92,5 +96,11 @@
         "@types/mocha": "^2.2.42",
         "@types/uuid": "^3.4.3"
     },
-    "extensionDependencies" : ["Pivotal.vscode-spring-boot", "redhat.java"]
+    "extensionDependencies": [
+        "Pivotal.vscode-spring-boot",
+        "redhat.java"
+    ],
+    "dependencies": {
+        "uuid": "^3.3.2"
```

Comment:
Why does the `@types/uuid` under `devDependencies` have a different version than the `uuid` under `dependencies`?